dowhy/causal_refuters/data_subset_refuter.py

```diff
@@ -60,7 +63,7 @@ def refute_once():
         sample_estimates = Parallel(
             n_jobs=self._n_jobs,
             verbose=self._verbose
-        )(delayed(refute_once)() for _ in range(self._num_simulations))
+        )(delayed(refute_once)() for _ in tqdm(range(self._num_simulations), colour='green', disable = not show_progress_bar, desc="Refuting Estimates: "))
```

Comment:
better to define "green" as a static variable inside causalrefuter class. and then call it here as CausalRefuter.Progress_Bar_Color